modules/nw-sriov-configure-exclude-topology-manager.adoc

```diff
@@ -0,0 +1,214 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+:_content-type: PROCEDURE
+[id="nw-sriov-configure-exclude-topology-manager_{context}"]
+= Excluding SR-IOV network topology for NUMA-aware scheduling 
+
+To exclude advertising the SR-IOV network resource's Non-Uniform Memory Access (NUMA) node to the Topology Manager, you can configure the `excludeTopology` specification in the `SriovNetworkNodePolicy` custom resource. Use this configuration for more flexible SR-IOV network deployments during NUMA-aware pod scheduling.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have configured the CPU Manager policy to `static`. For more information about CPU Manager, see the _Additional resources_ section.
+* You have configured the Topology Manager policy to `single-numa-node`.
+* You have installed the SR-IOV Network Operator.
+
+.Procedure
+
+. Create the `SriovNetworkNodePolicy` CR:
+
+.. Save the following YAML in the `sriov-policy-numa.yaml` file, replacing values in the YAML to match your environment:
```

Comment:
> in the `sriov-policy-numa.yaml` file

It isn't clear if the the `sriov-policy-numa.yaml` file is the the `SriovNetworkNodePolicy` CR you are creating or a different file.  If it is the same file, must the CR file named this?